### ISSUE TITLE
About ANCHOR_SCALES in config

### ISSUE THREAD

**Issue by chuckgu** [GeeshangXu/mask-rcnn-pytorch#2]
Anchor scales in no FPN mode are set as 

`ANCHOR_AREAS = 32 48 64`

`for i in range(len(anchor_areas)):`
    `   assert anchor_areas[i] % feat_stride == 0, "anchor area must be multiple of feat stride."`
    `   anchor_scales.append(anchor_areas[i] / feat_stride)`

So we can get [2.0, 3.0, 4.0] with feat stride 16.

In general Faster R CNN Source, anchor scales are set as [8,16,32]

I can't find any lines to match anchor scales in ur code to above general  anchor scales.

I think it can cause wrong sized anchor boxes in RPN.

or I think wrongly?

**Comment by GeeshangXu** [comment]
Sorry, I should mention that this config is for a custom dataset, this is a hyper-parameter you can adjust, if you want to train with COCO dataset, you can set it to `ANCHOR_AREAS = 128 256 512` as Faster R-CNN paper did. I will add a config.ini specific for COCO .
